lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_completecodefreeze_prechecks.rb

```diff
@@ -9,8 +9,10 @@ def self.run(params)
 
         UI.user_error!('This is not a release branch. Abort.') unless other_action.git_branch.start_with?('release/')
 
-        version = Fastlane::Helper::Android::VersionHelper.get_public_version
-        message = "Completing code freeze for: #{version}\n"
+        app = ENV['PROJECT_NAME'].nil? ? params[:app] : ENV['PROJECT_NAME']
```

Comment:
```suggestion
        app = params[:app]
```

Reply:
@AliSoftware Wouldn't applying this suggestion break clients that don't conform to these changes -> I left this in to allow for the environment variable to still be used

Reply:
Oof! Nevermind I missed that Fastlane handles this for me 👍

Reply:
yeah that's the whole point of the `env_name` attribute of the `ConfigItem` you define a couple of lines below in the action :)